src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,354 @@
+      "outline",
+      "rgb(255, 188, 25) solid 2px"
+    );
+  });
+});
+
+test.describe("Testing Help component properties", () => {
+  testData.forEach((ariaLabel) => {
+    test(`should check ariaLabel as ${ariaLabel}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest ariaLabel={ariaLabel} />);
+      await expect(helpComponent(page)).toHaveAttribute(
+        "aria-label",
+        ariaLabel
+      );
+    });
+  });
+
+  testData.forEach((className) => {
+    test(`should check className as ${className}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest className={className} />);
+
+      await containsClass(helpComponent(page), className);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children}`, async ({ mount, page }) => {
+      await mount(<Help> {children} </Help>);
+
+      await helpComponent(page).hover();
+      const tooltip = getDataElementByValue(page, "tooltip");
+      await expect(tooltip).toBeVisible();
+      await expect(tooltip).toHaveText(children);
+    });
+  });
+
+  testData.forEach((helpId) => {
+    test(`should check id as ${helpId}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest helpId={helpId} />);
+      await expect(helpComponent(page)).toHaveId(helpId);
+    });
+  });
+
+  ["https://carbon.sage.com", "https://www.google.com"].forEach((hrefLink) => {
+    test(`should check href link as ${hrefLink}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest href={hrefLink} />);
+      await expect(helpComponent(page)).toHaveAttribute("href", hrefLink);
+    });
+  });
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when isFocused prop is passed as ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest isFocused={boolVal}>{tooltipText}</HelpComponentTest>
+      );
+      const tooltip = getDataElementByValue(page, "tooltip");
+      if (boolVal === true) {
+        await expect(tooltip).toBeVisible();
+      } else {
+        await expect(tooltip).not.toBeVisible();
+      }
+    });
+  });
+
+  [-1, 0].forEach((tabIndex) => {
+    test(`should check tabIndex as ${tabIndex}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest tabIndex={tabIndex} />);
+
+      if (tabIndex === -1) {
+        await page.keyboard.press("Tab");
+        await expect(helpComponent(page)).not.toBeFocused();
+      } else {
+        await page.keyboard.press("Tab");
+        await expect(helpComponent(page)).toBeFocused();
+      }
+    });
+  });
+
+  colors.forEach(([names, color]) => {
+    test(`should check tooltipBgColor as ${names}`, async ({ mount, page }) => {
+      await mount(
+        <HelpComponentTest tooltipBgColor={color} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      const tooltip = getDataElementByValue(page, "tooltip");
```

Comment:
**suggestion: (non-blocking)** as above, it would be better to add a space to separate the mount from the assertion to follow the same pattern as with the other test cases.